content/articles/how-to-use-symbolic-math's-toolbox-in-matlab-to-analyze-the-fourier-series/index.md

```diff
@@ -0,0 +1,281 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /how-to-use-symbolic-math's-toolbox-in-matlab-to-analyze-the-fourier-series/
```

Comment:
For the article directory, please avoid using `'`.

The directory should always be in lowercase, separated by `-`.

If possible, try to make the path a little smaller.

Reply:
@srishilesh how can I change the folder name?

Reply:
You would have to rename the folder. And, also edit the same in this file too.

Reply:
> For the article directory, please avoid using `'`.
> 
> The directory should always be in lowercase, separated by `-`.
> 
> If possible, try to make the path a little smaller.

@srishilesh, it now looks good. The folder name is changed. The tag added. Any more recommendations?